code/modules/client/preferences_savefile.dm

```diff
@@ -266,12 +293,6 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 
 	scars_index = rand(1,5)
 
-	if(!S["features["mcolor"]"] || S["features["mcolor"]"] == "#000")
-		WRITE_FILE(S["features["mcolor"]"]	, "#FFF")
-
-	if(!S["feature_ethcolor"] || S["feature_ethcolor"] == "#000")
-		WRITE_FILE(S["feature_ethcolor"]	, "9c3030")
-
```

Comment:
These were removed because this shouldn't even be in load_character, and its repeated down below in the sanitize section, and the first one wasn't even correct.